src/docs/product/releases/health/release-details.mdx

```diff
@@ -0,0 +1,32 @@
+---
+title: Release Details
+sidebar_order: 50
+description: "Learn more about the details of individual releases."
+---
+
+The Release Details page focuses on an individual release. Elements of the release are presented in visual trends for crashes and sessions, specifics regarding each issue, and commit author breakdowns.
+
+![View of the release details page showing an overview graph, files changed, and other details.](release_details.png)
+
+Hover over specific dates and times in the overview graph for a quick perspective of events that crashed, behaved abnormally, errored, or are healthy.
+
+Crashed
```

Comment:
Can we move these definitions into the "Sessions" section on the index page? I feel like they are more canonical information and not specific to the details page. For instance, those states will also show up on the release index page, and they contribute to the definition of what a "crash free user" is.

Reply:
Yes. Will do.

Reply:
I will also combine the definitions that overlap. Do we need more for these definitions, perhaps some examples?